test/Swashbuckle.AspNetCore.SwaggerGen.Test/SchemaGenerator/JsonSerializerSchemaGeneratorTests.cs

```diff
@@ -670,15 +671,55 @@ public void GenerateSchema_SupportsOption_UseInlineDefinitionsForEnums()
             Assert.NotNull(schema.Enum);
         }
 
+        [Fact]
+        public void TypeWithNullableContextAnnotated_IsAnnotated()
+        {
+            const string Name = "System.Runtime.CompilerServices.NullableContextAttribute";
+
+            var nullableContext = typeof(TypeWithNullableContextAnnotated)
+                .GetCustomAttributes()
+                .FirstOrDefault(attr => string.Equals(attr.GetType().FullName, Name));
+
+            Assert.NotNull(nullableContext);
+
+            var flag = nullableContext?.GetType().GetField("Flag")?.GetValue(nullableContext);
+
+            Assert.Equal((byte)2, flag);
+        }
+
+        [Fact]
+        public void TypeWithNullableContextNotAnnotated_IsNotAnnotated()
```

Comment:
Are these two tests just "help you develop" tests? They don't seem to test any part of Swashbuckle, but just assumptions/understanding of nullable annotations.

Reply:
They were intended as sanity checks for the test types, so that we know if our assumptions are suddenly not correct anymore. It can change invisibly and without error or notice if we rewrite the types, or if the compiler behavior changes. 

We have no direct control over how the compiler places its attributes, but for the test suite to be effective we need the attributes to be placed in a certain way.

I should probably add a comment in each test to mention this.

Reply:
Ok cool, that makes sense. Yeah some comments to that effect would be good.